setup.py

```diff
@@ -58,6 +58,10 @@
     'cc-plugin-imos>=1.3.0',
     'fiona>=1.8.8',
     'matplotlib>=3.0.3',
+    'numpy>=1.13.0',
```

Comment:
These dependencies are not required by *this* package, so don't need to be in here. The are either "dependencies of dependencies" or provided by the dependencies anyway. We've found through trial and error that keeping the requirements minimal here saves a lot of conflicts and "fixup" PRs later.

Reply:
OK - I include that because of my pip experience with circular dependencies and thought that failing towards the leaf was best.

On the funny side, this is a fixup PR because of that same reason! Maybe we need a jenkins job just for dependencies check then?